martin/src/fonts/mod.rs

```diff
@@ -301,44 +300,43 @@ fn parse_font(
             .replace_all(name.as_str(), " ")
             .to_string();
 
-        match fonts.entry(name) {
-            Entry::Occupied(v) => {
+        if let Some(v) = fonts.get(&name) {
```

Comment:
dashmap support the entry api - why did you switch to the get+insert, with a potential for a race condition?

Reply:
I somehow thought that the entry api did not exist.
Seems baffeling in hindsight 🤷🏻‍♂️